plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstanceProvider.java

```diff
@@ -142,6 +145,7 @@ public DockerInstanceProvider(DockerConnector docker,
         this.supportedRecipeTypes = Sets.newHashSet(DOCKER_FILE_TYPE, DOCKER_IMAGE_TYPE);
         this.projectFolderPath = projectFolderPath;
         this.snapshotUseRegistry = snapshotUseRegistry;
+        this.memorySwapMultiplier = memorySwapMultiplier == -1 ? -1 : memorySwapMultiplier + 1;
```

Comment:
@riuvshin can you please add a comment explaining the `memorySwapMultiplier + 1` because it is really unclear from the context, and it took some time for me to figure out the reason of adding `1` to the multiplier